psi4/src/CMakeLists.txt

```diff
@@ -127,7 +127,6 @@ message(STATUS "Psi4 rpath: ${psi4_RPATH}")
 set_target_properties(core PROPERTIES PREFIX "${PYTHON_MODULE_PREFIX}"  # for python module
                            OUTPUT_NAME core
                            EXPORT_NAME core
-                           SUFFIX ".so"  # for python module
-                           #SUFFIX "${PYTHON_MODULE_EXTENSION}" # for python module
+                           SUFFIX "${PYTHON_MODULE_EXTENSION}" # for python module
```

Comment:
I think we had hardcoded the `.so` rather than something like `cpython-36m-x86_64-linux-gnu.so` for the benefit of plugins, so they could find the `core.so` to link to in the ordinary shared library manner. I bet we can be cleverer with this now so the variable file name won't be a problem. So this is just a flag so I remember to test.

Reply:
`ENABLE_PLUGIN_TESTING` is a good start (and enough to merge). Then we wait for the approved plugins and @fevangelista's Forte to uncover problems. :-)